microcosm_flask/conventions/relation.py

```diff
@@ -205,11 +204,10 @@ def search(**path_data):
         search.__doc__ = "Search for {} relative to a {}".format(pluralize(ns.object_name), ns.subject_name)
 
 
-def configure_relation(graph, ns, mappings, path_prefix=""):
+def configure_relation(graph, ns, mappings):
     """
     Register relation endpoint(s) between two resources.
 
     """
-    ns = Namespace.make(ns, path=path_prefix)
```

Comment:
Stop trying to generate `Namespace` arguments on-the-fly.

This ability was added to transition to the `Namespace` class, which is now widely used.